src/views/wiki/edition/RouteEditionView.vue

```diff
@@ -194,6 +189,14 @@ export default {
       }
     },
 
+    onGeometryUpdate(to, from) {
```

Comment:
forgot this earlier, but this method is used in many edition views. Could be put in a mixin?